drivers/strm/meta.go

```diff
@@ -10,6 +10,7 @@ type Addition struct {
 	SiteUrl         string `json:"siteUrl" type:"text" required:"false" help:"The prefix URL of the strm file"`
 	FilterFileTypes string `json:"filterFileTypes" type:"text" default:"strm" required:"false" help:"Supports suffix name of strm file"`
 	EncodePath      bool   `json:"encodePath" default:"true" required:"true" help:"encode the path in the strm file"`
+	LocalModel      bool   `json:"local_model" default:"false" help:"enable local mode"`
```

Comment:
这个 json 字段命名和上面的风格好像有点不匹配
```suggestion
	LocalModel      bool   `json:"localModel" default:"false" help:"enable local mode"`
```